helpers/PassConverter.js

```diff
@@ -0,0 +1,27 @@
+const crypto = require('crypto');
+
+
+module.exports = class PassConverter {
+    // hashing password and genereate salt 
+   static hash = async (password) => {
+        return new Promise((resolve, reject) => {
```

Comment:
Please check is crypto providing promisify function and use it to remove the callback